keys.go

```diff
@@ -80,6 +80,7 @@ type Key struct {
 	PurgeAllowedFrom    *time.Time  `json:"purgeAllowedFrom,omitempty"`
 	PurgeScheduledOn    *time.Time  `json:"purgeScheduledOn,omitempty"`
 	DualAuthDelete      *DualAuth   `json:"dualAuthDelete,omitempty"`
+	Search              string      `json:"search,omitempty"`
```

Comment:
I don't understand why we are adding and extra field to the Key data that comes from the service. I don't see it in any of the documentation either.